src/Storage/Storage/Blob/Cmdlet/GetAzureStorageBlob.cs

```diff
@@ -34,6 +34,9 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     /// <summary>
     /// list azure blobs in specified azure container
     /// </summary>
+    [GenericBreakingChange("Leading and trailing slashes on the blob name will not be trimmed when getting a single blob",
+       OldWay = "Leading and trailing slashes on the blob name are trimmed",
```

Comment:
Could we don't add oldway/newway?
This change impact might be small, so don't want to make the warning too long.

Reply:
Yes. Deleted.